.github/workflows/integration-test.yml

```diff
@@ -88,5 +111,5 @@ jobs:
         run: |
           echo "check if opensearch is ready"
           curl -XGET https://localhost:9200 -u 'admin:admin' -k
-          node .\test\run_jest_tests.js --config .\test\jest.config.server.js --testPathIgnorePatterns saml_auth.test.ts --testPathIgnorePatterns saml_multiauth.test.ts
+          node .\test\run_jest_tests.js --runInBand --detectOpenHandles --forceExit --config .\test\jest.config.server.js --testPathIgnorePatterns saml_auth.test.ts --testPathIgnorePatterns saml_multiauth.test.ts
```

Comment:
I ran into the same issue described in this SO post: https://stackoverflow.com/a/70432217

Added `--runInBand --detectOpenHandles --forceExit` for debugging and to exit the tests. I'm not sure how else to resolve the test issue at this time.

```
Jest did not exit one second after the test run has completed.

This usually means that there are asynchronous operations that weren't stopped in your tests. Consider running Jest with `--detectOpenHandles` to troubleshoot this issue.
```

Reply:
aahh.. I ran into this too and I think these args resolved it on my end too. Seems like a good fix to me.